content/en/docs/v3.5/downgrades/downgrade_3_5.md

```diff
@@ -0,0 +1,276 @@
+
+Before beginning, download the latest release of etcd 3.4, and make sure its version is >= v3.4.32.
+
+#### Mixed versions
+
+While downgrading, an etcd cluster supports mixed versions of etcd members, and operates with the protocol of the lowest common version. The cluster is considered downgraded once any of its members is downgraded to version 3.4. Internally, etcd members negotiate with each other to determine the overall cluster version, which controls the reported version and the supported features.
+
+#### Limitations
+
+Note: If the cluster only has v3 data and no v2 data, it is not subject to this limitation.
+
+If the cluster is serving a v2 data set larger than 50MB, each newly downgraded member may take up to two minutes to catch up with the existing cluster. Check the size of a recent snapshot to estimate the total data size. In other words, it is safest to wait for 2 minutes between downgrading each member.
+
+For a much larger total data size, 100MB or more , this one-time process might take even more time. Administrators of very large etcd clusters of this magnitude can feel free to contact the [etcd team][etcd-contact] before downgrading, and we'll be happy to provide advice on the procedure.
+
+#### Rollback
+
+If any member has been downgraded to v3.4, the cluster will be downgraded to v3.4, and the cluster and its operations will be "v3.4". You would need to follow the [Upgrade etcd from 3.4 to 3.5](../../upgrades/upgrade_3_5/) instructions to rollback.
+
+Please [download the snapshot backup](../../op-guide/maintenance/#snapshot-backup) to make downgrading the cluster possible even after it has been completely downgraded.
+
+### Downgrade procedure
+
+This example shows how to downgrade a 3-member v3.5 etcd cluster running on a local machine.
+
+#### Step 1: check downgrade requirements
+
+Is the cluster healthy and running v3.5.x?
+
+```bash
+etcdctl --endpoints=localhost:2379,localhost:22379,localhost:32379 endpoint health
+<<COMMENT
+localhost:2379 is healthy: successfully committed proposal: took = 2.118638ms
+localhost:22379 is healthy: successfully committed proposal: took = 3.631388ms
+localhost:32379 is healthy: successfully committed proposal: took = 2.157051ms
+COMMENT
+
+curl http://localhost:2379/version
+<<COMMENT
+{"etcdserver":"3.5.0","etcdcluster":"3.5.0"}
+COMMENT
+
+curl http://localhost:22379/version
+<<COMMENT
+{"etcdserver":"3.5.0","etcdcluster":"3.5.0"}
+COMMENT
+
+curl http://localhost:32379/version
+<<COMMENT
+{"etcdserver":"3.5.0","etcdcluster":"3.5.0"}
+COMMENT
+```
+
+#### Step 2: download snapshot backup from leader
```

Comment:
I think it's overkill to instruct how to take snapshot from leader. Followers should not be much behind and it complicates the procedure.

Reply:
removed